### ISSUE TITLE
Finchley-Snapshot Netflix-Archaius dependency: commons-configuration is runtime

### ISSUE THREAD

**Issue by ramato-procon** [spring-cloud/spring-cloud-netflix#2886]
I have recently updated from SpringBoot 1.5.10 to 2.0.1 and Spring Cloud Finchley (2.0.0.BUILD-20180426.110507-1967).
During maven compile I get following error: 

> COMPILATION ERROR :
> cannot access org.apache.commons.configuration.AbstractConfiguration
> class file for org.apache.commons.configuration.AbstractConfiguration not found

I'm using the Hystrix ConfigurationManager Class to configure Hystrix at SpringBoot startup.

```

ConfigurationManager.getConfigInstance().setProperty(
				"hystrix.command.default.execution.isolation.thread.timeoutInMilliseconds",
				config.getHystrix().getTimeoutInMilliseconds());
```

The problem is the scope for commons-configuration is "runtime", but it should "compile"

Maybe this happend with this commit:
https://github.com/spring-cloud/spring-cloud-netflix/commit/73fbec166ddf591cdfa8aebce59d6279f12cad76#diff-c70ec716b8fb58d81605b2470bbc354e

Maybe earlier, but with SpringBoot 1.5.10 and Spring Cloud Edgware SR3 it was working.

**Comment by spencergibb** [comment]
Closed via 2e12d9b473da10351c298a991ddb888ec977152f

@ramato-procon can you give snapshots another try shortly?

**Comment by ramato-procon** [comment]
@spencergibb Working now. Thank you!
